autoload/lsp/internal/completion/documentation.vim

```diff
@@ -65,43 +68,113 @@ endfunction
 
 function! s:show_floating_window(event, managed_user_data) abort
     let l:completion_item = a:managed_user_data['completion_item']
-    let l:documentation = get(l:completion_item, 'documentation', '')
-    if type(l:documentation) == type({}) && has_key(l:documentation, 'value')
-        let l:documentation = l:documentation['value']
+
+    " Ignore if documentation does not exists.
+    if !has_key(l:completion_item, 'documentation')
```

Comment:
should this also check for detail? I'm wondering if we should show detail if documentation doesn't exist. i don't have a strong preference for this. Might partially solve this issue https://github.com/prabirshrestha/vim-lsp/issues/1044

Reply:
currently seems like we need to have documentation in order to show details + doc. not sure how vs-code does it.

Reply:
I've improved to resolve `documentation/detail`.